packages/client/ui/react-ui/src/components/auth/PasskeyPrompt.tsx

```diff
@@ -0,0 +1,266 @@
+import { Button, Dialog, Transition } from "@headlessui/react";
+import type React from "react";
+import { Fragment, type ReactNode } from "react";
+import type { UIConfig } from "@crossmint/common-sdk-base";
+
+import FingerprintIcon from "../../icons/fingerprint";
+import PasskeyIcon from "../../icons/passkey";
+import PasskeyPromptLogo from "../../icons/passkeyPromptLogo";
+import { PoweredByCrossmint } from "../common/PoweredByCrossmint";
+import { classNames } from "@/utils/classNames";
+
+const primaryButtonStyles = (props: { appearance?: UIConfig }): React.CSSProperties => ({
```

Comment:
nit: IMO this is info that's less important than being able to grok the component, so I'd typically put this stuff at the bottom of the file a bit more out of the way

Reply:
fixed